tests/test_001_to_from.py

```diff
@@ -368,7 +369,28 @@ def test_knmi_collection_from_grid():
 
 def test_waterinfo_from_dir():
     path = "./tests/data/2023-waterinfo-test"
-    hpd.read_waterinfo(path)
+    hpd.read_waterinfo(file_or_dir=path)
+
+
+def test_waterinfo_ddlpy():
+    grootheid_code = "WATHTE"
+    locatie = "SCHOONHVN"
+    tmin = dt.datetime(2024, 1, 1)
+    tmax = dt.datetime(2024, 1, 2)
+    hpd.WaterlvlObs.from_waterinfo(
+        grootheid_code=grootheid_code, locatie=locatie, tmin=tmin, tmax=tmax
+    )
+
+
+def test_waterinfo_ddlpy_extent():
+    grootheid_code = "WATHTE"
+    tmin = dt.datetime(2024, 1, 1)
```

Comment:
dt.datetime or Timestamp?

Reply:
I removed the datetime in favor of Timestamp